lib/user_space/ugetopts.c

```diff
@@ -112,6 +112,7 @@ int m0_getopts(const char *progname, int argc, char *const *argv,
 		/* -W is reserved by POSIX.2 and used by GNU getopts as a long
 		   option escape. */
 		M0_ASSERT(opts[i].go_opt != 'W');
+		M0_ASSERT(strchr(optstring, opts[i].go_opt) == NULL);
```

Comment:
Some programs accept the same args multiple times.

So, I assume we are avoiding that, because it is not a usual way.

Reply:
Thank you, @nikitadanilov . Understood.